ApeWorX/silverback
# chore: update pin for ApePay

### What I did

- Pinned `apepay` version to `>=0.3.2,<1` in `install_requires` to maintain compatibility and stability.

### How I did it

1. Updated `install_requires` to restrict `apepay` to a compatible version with the apepay [#115](https://github.com/ApeWorX/ApePay/pull/115).

### How to verify it

1. Verify that `apepay` installs the correct version by running `pip install . -U`.
